app/controllers/cell_controller.py

```diff
@@ -1,16 +1,26 @@
-from app.model import ArchiveOperator, CellMeta
-from app.aio import ArchiveExporter
+from app.model import ArchiveOperator
+from app.aio import ArchiveExporter, GAReader
 from app.archive_cell import ArchiveCell
-from flask import request
+from flask import request, jsonify
 import pandas as pd
-from app.archive_constants import (LABEL, DEGREE, SLASH,
-                                       CELL_LIST_FILE_NAME, TEST_TYPE, FORMAT)
-
+from app.archive_constants import (LABEL, SLASH,
+                                   CELL_LIST_FILE_NAME, TEST_TYPE, FORMAT)
+import uuid
+import threading
 # Routes
+"tracker -> msg"
+global status
+status = {}
+"tracker -> id"
+global source
+source = {}
+
+# STATUS OPTIONS:
+# STARTED, IN_PROGRESS, FINISHED
```

Comment:
Would it make sense to move these to `archive_constant.py` as a constant or enum?